### ISSUE TITLE
Support for RabbitMQ 3.10.x

### ISSUE THREAD

**Issue by mymasse** [pivotal-cf/cf-rabbitmq-release#582]
Any plans on adding support for the 3.10 release stream. 3.10.0 as been release close to 3 weeks ago and is now at 3.10.2

**Comment by coro** [comment]
Yes! The next release (which is building in the pipeline right now) will have 3.10.5.
